### PR TITLE
Add new maintainer

### PR DESCRIPTION
**Pull request by bollwyvl**
For #9.

@conda-forge-admin please merge so we can get this one back on track!

cc @carlodri

### CODE REVIEW

**Comment by carlodri** [comment]
@conda-forge/core or @pmlandwehr can someone please merge so that we can give a new life to this feedstock?

**Comment by carlodri** [comment]
@bollwyvl you can add me as well (see my [suggested change](https://blog.github.com/changelog/2018-10-16-suggested-changes/) below)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -52,3 +52,4 @@ about:
 extra:
   recipe-maintainers:
     - pmlandwehr
+    - bollwyvl
```
Comment (carlodri) [inline recipe/meta.yaml:55]:
```suggestion
    - bollwyvl
    - carlodri
```

**Comment by bollwyvl** [comment]
Just on my phone right now, but will add you as soon as I can.

**Comment by carlodri** [comment]
thanks @isuruf!

**Comment by bollwyvl** [comment]
Cheers, thanks all!

**Pull request merged**